relay-event-normalization/src/event.rs

```diff
@@ -202,6 +202,9 @@ fn normalize(event: &mut Event, meta: &mut Meta, config: &NormalizationConfig) -
     // Check for required and non-empty values
     let _ = schema::SchemaProcessor.process_event(event, meta, ProcessingState::root());
 
+    // Set required attributes to default values if not present.
+    normalize_default_required_attrs(event);
```

Comment:
Let's move this lower, as the event may still be rejected after this point. There are some required attribute normalization on line 255, that could be a good place.

Reply:
Thanks for the tip! Will do